### DISCUSSION TITLE
Reminder: ToIP WG Contribution Policies

### DISCUSSION THREAD

**Discussion by jorydotcom** [trustoverip/trust-spanning-protocol#31] [General]
This is a general reminder to the community, as may be beneficial from time to time. 

Please note that Working Group discussions and contributions must be made under the intellectual property policy of the Working Group as specified in the Working Group charter. Additionally, contributions should be made to the electronic workspace(s) selected by the WG (GitHub, Groups.io Mailing discussion lists, Confluence, and others) versus any site that might be owned by a member. Finally, any summaries of WG activities should give appropriate reference to the WG including the inclusion of any copyright attribution information.

Thank you for your participation and cooperation!

**Comment by talltree** [comment]
Thanks @jorydotcom. As a reminder to all TSPTF members, we operate under the IPR policy of the Technology Stack Working Group as listed at the bottom of [its home page](https://wiki.trustoverip.org/display/HOME/Technology+Stack+Working+Group). The essence is:

- Copyright mode: [Creative Commons Attribution 4.0](http://creativecommons.org/licenses/by/4.0/legalcode). 
- Patent mode: W3C Mode (based on the [W3C Patent Policy](http://www.w3.org/Consortium/Patent-Policy-20040205)).
- Source code: Apache 2.0, available at http://www.apache.org/licenses/LICENSE-2.0.html.

Any contribution that is incorporated into one or more of our final deliverable(s) will be licensed by the ToIP Foundation under those IPR terms.

This doesn't mean you can't reference work published elsewhere in our meetings or discussions. And if that work is a publicly available open standard, we can always choose to normatively reference it. But as Jory points out, if you intend to make a contribution here of work that was developed externally, is not a publicly available open standard, and for which you own the IPR rights, you need to place a copy of your contribution in our electronic workspaces here (such as this GitHub repo), and in doing so agree that it will be licensed as described above.

**Comment by JFleenor** [comment]
Thank you Jory and Drummond,  I'd like to add that per the JDF documents signed by members contributions and not just what has been placed in writing, but also included what is expressed orally in our meetings.   The exact wording from the JDF Member Agreement appendix A is as follows:

5.2. “Contribution” means any material, including any modifications or additions to an existing work, that the Steering Member, General Member, or Contributor submits for inclusion in the Draft Deliverable or Approved Deliverable, which is included in the Draft Deliverable or Approved Deliverable. For the purposes of this definition, “submit” means any form of electronic, oral, or written communication for the purpose of discussing and improving the Draft Deliverable or Approved Deliverable, but excluding communication that the Steering Member, General Member, or Contributor conspicuously designate in writing as not a contribution.
